appveyor.yml

```diff
@@ -26,6 +30,7 @@ matrix:
 platform:
     -x64
 
+os: Visual Studio 2015 Update 2
```

Comment:
@sosey - Any reason to change the image?  I think imexam should be fine with the default one.

Reply:
otherwise, the org "spacetelescope" need to have the image added by appveyor as it's not available any more by default.

Reply:
yah, I asked to have it added by appveyor,  I was trying to remember last night why the image was in there, it might have been for the xpa build

Reply:
yes, but builds were passing before and I don't see anything in this PR and indicates that it would fail with the default.

(also I suspect the changes to the helpers here is accidental?)

Reply:
yes, this is an old pr that I mucked up on a merge, I'm working on fixing it but might not get to it until tomorrow